common/src/main/java/com/wynntils/gui/screens/settings/WynntilsBookSettingsScreen.java

```diff
@@ -380,7 +380,8 @@ private void reloadConfigurableButtons() {
     }
 
     public boolean configOptionContains(ConfigHolder configHolder) {
-        return StringUtils.containsIgnoreCase(configHolder.getDisplayName(), searchWidget.getTextBoxInput());
+        return !searchWidget.getTextBoxInput().isEmpty()
+                && StringUtils.containsIgnoreCase(configHolder.getDisplayName(), searchWidget.getTextBoxInput());
```

Comment:
What does this fix?

Reply:
everything being treated as searched/highlighted